src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -155,15 +157,13 @@ public final static class SimpleEntityDataInfo {
 		final String codeName;
 		final Class<? extends Entity> c;
 		final boolean isSupertype;
+		final Kleenean allowSpawning;
 		
-		SimpleEntityDataInfo(final String codeName, final Class<? extends Entity> c) {
-			this(codeName, c, false);
-		}
-		
-		SimpleEntityDataInfo(final String codeName, final Class<? extends Entity> c, final boolean isSupertype) {
+		SimpleEntityDataInfo(String codeName, Class<? extends Entity> c, boolean isSupertype, Kleenean allowSpawning) {
```

Comment:
Is there still a constructor with supertype but without allowSpawning?

Reply:
These are private constructors so I thought it would be safe to remove them.

Reply:
Ah true, nevermind then